lib/config.js

```diff
@@ -1,44 +1,51 @@
-var defaults = require('lodash.defaults');
-var compact = require('lodash.compact');
-var map = require('lodash.map');
-var fs        = require('fs');
-var path      = require('path');
-var osHomedir = require('os-homedir');
+'use strict';
 
-var config = null;
+const defaults = require('lodash.defaults');
```

Comment:
Not unless that is actually forced by the linter. For example, the golang's linter(gfmt) automatically does this. For now, this is not the priority.